Formula/molten-vk.rb

```diff
@@ -143,6 +143,7 @@ def install
     include.install Dir["Package/Release/MoltenVKShaderConverter/include/" \
                         "MoltenVKShaderConverter"]
 
+    inreplace "MoltenVK/icd/MoltenVK_icd.json", "\"./libMoltenVK.dylib\"", "\"../../../lib/libMoltenVK.dylib\""
```

Comment:
Is this issue submitted upstream? What is it actually fixing?

Reply:
I'm not sure if an upstream submission would really be appropriate, as this is specific to installing system-wide, which the upstream package doesn't provide any support for (we do it all manually in the formula).

The issue is that the actual driver lib is installed in `{formula_prefix}/lib/libMoltenVK.dylib` (fairly sensibly), while the `icd.json` is installed in `{formula_prefix}/share/vulkan/icd.d/MoltenVK_icd.json` (also fairly sensible), and the `library_path` property needs to point to the driver lib relative to the dir containing the json. The upstream package just gives `./libMoltenVK.dylib`, which is appropriate when packaging MoltenVK in a .app bundle with the dylib and json files right next to each other (which is the main use-case they target).

Reply:
Can we use `Pathname#relative_path_from` to make this understandable without having to check the directory structure manually (`grep relative_path_from` for examples)?

Reply:
It can indeed! Pushed.